pyflakes/checker.py

```diff
@@ -746,6 +746,9 @@ def handleNodeLoad(self, node):
             # the special name __path__ is valid only in packages
             return
 
+        if name == '__module__' and isinstance(self.scopeStack[-1], ClassScope):
```

Comment:
`self.scopeStack[-1]` is property `self.scope`

Reply:
Thank you!